backend/src/routes/Meeting.ts

```diff
@@ -404,4 +404,98 @@ router.put('/withdraw', async (req: Request, res: Response, next: NextFunction)
         .catch(err => next(err));
 });
 
+/**
+ * Update pause meeting
+ * 
+ * @params  meetingAddress  The _id to look for.
+ * 
+ * @returns Meeting
+ */
+router.put('/pause', async (req: Request, res: Response, next: NextFunction) => {
+    Models.Item
+        .updateOne(
+            { _id: req.body['meetingAddress'], type: ModelType.MEETING },
+            { $set: { 'data.isPaused': true } },
+            { safe: true, upsert: true }
+        )
+        .then(meeting => {
+            res
+                .status(OK)
+                .json(meeting);
+        })
+        .catch(err => next(err));
+});
+
+/**
+ * Add proposal to database
+ * 
+ * @params  meetingAddress  The _id to look for.
+ * 
+ * @returns Meeting
+ */
+router.put('/proposal/add', async (req: Request, res: Response, next: NextFunction) => {
+    Models.Item
+        .updateOne(
+            { _id: req.body['meetingAddress'], type: ModelType.MEETING },
+            {
+                $push: { 'data.proposals': req.body['proposal'] },
+            },
+            { safe: true, upsert: true }
+        )
+        .then(meeting => {
+            res
+                .status(OK)
+                .json(meeting);
+        })
+        .catch(err => next(err));
+});
+
+/**
+ * Vote on proposal (increase vote count)
+ * 
+ * @params  meetingAddress  The _id to look for.
+ * 
+ * @returns Meeting
+ */
+router.put('/proposal/vote', async (req: Request, res: Response, next: NextFunction) => {
+    Models.Item
+        .updateOne(
+            { _id: req.body['meetingAddress'], type: ModelType.MEETING },
+            {
+                $push: { 'data.proposals': req.body['proposal'] },
```

Comment:
this one currently duplicates the proposal in the database (with different vote counts) after voting on it. I am still looking for the best solution, so you can skip it for now.